### PR TITLE
Rankings page 

### PR DESCRIPTION
**Pull request by rapperdinesh**
---

**What's this PR do? (related issue if exists)**
Display Rankings Table to User
**Any context you want to provide on the implementation?**
localhost:3000/rankings  will render you this page
**Screenshots? (necessary if an UI/UX change)**
https://pasteboard.co/HTocJMM.png
https://pasteboard.co/HTodfIu.png

### CODE REVIEW

**Comment by rapperdinesh** [comment]
Link for screenshot of page 
1. https://pasteboard.co/HTocJMM.png
2.  https://pasteboard.co/HTodfIu.png

**Comment by Vishal1541** [comment]
@rapperdinesh 
1. Don't make the footer fixed.
2. Change the ** IIT Mandi Official Coding Contest sites** to ** IIT Mandi Official Online Judge** and try placing it just below the logo.
3. Try placing the dropdown buttons inline with the logo.
4. Make this navbar the header which will be common to all the pages and reduce it's size.
5. Make the header fixed.

File: views/partials/head.ejs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      text-decoration: none;
+      margin: 2px 2px;
+    }
+
+  .fa:hover {
+      opacity: 0.9;
+  }
+
+  .fa-linkedin {
+    /* background: #007bb5; */
```
Comment (Vishal1541) [inline views/partials/head.ejs]: It's a good practice to always try to remove unwanted/ unused code :smiley:

**Comment by rapperdinesh** [comment]
That footer code was from my kwoc project which i added there that's why it contained those unused lines :)
And Thanks for the tip :)

**Pull request merged**